app/views/thread_header.html

```diff
@@ -1,10 +1,17 @@
+<div class="bu-QuestionMeta">
+  <span>
+    {{ displayable_metadata.handset_type }}
+    {# HTML comments are added below to prevent unwanted whitespace #}
+    {{ displayable_metadata.os_version }}<!--
+    {% if (displayable_metadata.handset_type or displayable_metadata.os_version) and displayable_metadata.operator %}
+      -->, <!--
+    {% endif %}
+    -->
```

Comment:
Nunjucks has [whitespace control](https://mozilla.github.io/nunjucks/templating.html#whitespace-control) so this can be done in a simpler way:
```jinja
{# Notice the dashes for whitespace control #}
{%- if (displayable_metadata.handset_type or displayable_metadata.os_version) and displayable_metadata.operator -%}
```
This way, we can ditch the `display: flex;` and `align-items: center;`